src/disco/reducers/discoResults.js

```diff
@@ -1,11 +1,95 @@
-import { LOAD_DISCO_RESULTS } from 'disco/constants';
+/* @flow */
+import invariant from 'invariant';
 
-export default function discoResults(state = [], { type, payload }) {
-  switch (type) {
+import type { AddonType, ExternalAddonType } from 'core/types/addons';
+
+export const GET_DISCO_RESULTS: 'GET_DISCO_RESULTS' = 'GET_DISCO_RESULTS';
```

Comment:
could this be called FETCH_DISCO_RESULTS... it's more consistent with other get/fetch names that I am seeing

Reply:
In a follow-up, yep. I tried to not change any code here, just moved things and added flow types.